### PR TITLE
docs(README): add index

### PR DESCRIPTION
**Pull request by bhavya9107**
**What kind of change does this PR introduce?**
Make doc efficient to be used.

**Did you add tests for your changes?**
N/A

**Does this PR introduce a breaking change?**
N/A

**What needs to be documented once your changes are merged?**
N/A

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,6 +43,23 @@
...
+5. [Support](#support)
+6. [Core Team](#core-team)
+7. [Sponsoring](#sponsoring)
+8. [Premium Partners](#premium-partners)
+9. [Other Backers and Sponsors](#other-backers-and-sponsors)
+10. [Gold Sponsors](#gold-sponsors)
+11. [Silver Sponsors](#silver-sponsors)
+12. [Bronze Sponsors](#bronze-sponsors)
+13. [Backers](#backers)
+14. [Thanks to](#thanks-to)
```
Comment (rishabh3112) [inline README.md]: Use `Special thanks` instead of `thanks to` or something similar.

**Comment by bhavya9107** [comment]
Please review and reply.

File: README.md [hunk 2]
```diff
@@ -43,6 +43,23 @@
...
+5. [Support](#support)
+6. [Core Team](#core-team)
+7. [Sponsoring](#sponsoring)
+8. [Premium Partners](#premium-partners)
+9. [Other Backers and Sponsors](#other-backers-and-sponsors)
+10. [Gold Sponsors](#gold-sponsors)
+11. [Silver Sponsors](#silver-sponsors)
+12. [Bronze Sponsors](#bronze-sponsors)
+13. [Backers](#backers)
+14. [Special Thanks To](#special-thanks-to)
```
Comment (rishabh3112) [inline README.md]:
```suggestion
14. [Special Thanks](#special-thanks-to)
```

**Comment by sokra** [comment]
Thanks

**Pull request merged**